e2e/support/helpers/e2e-embedding-helpers.js

```diff
@@ -262,6 +262,12 @@ export function createPublicDashboardLink(dashboardId) {
   return cy.request("POST", `/api/dashboard/${dashboardId}/public_link`, {});
 }
 
+/**
+ * @param {Object} options
+ * @param {string} options.url
+ * @param {Object} options.qs
+ * @param {Function} [options.onBeforeLoad]
```

Comment:
the function is optional, but typescript doesn't understand that without the jsdoc